pkg/kv/kvserver/replica_app_batch.go

```diff
@@ -410,12 +415,23 @@ func (b *replicaAppBatch) runPostAddTriggersReplicaOnly(
 		// going to bother with a long-running migration.
 		apply := !looselyCoupledTruncation || res.RaftExpectedFirstIndex == 0
 		if apply {
+			// TODO(pavelkalinnikov): counting bytes may be unreliable because 0 bytes
+			// does not mean 0 files. Make the interface more robust.
```

Comment:
I don't think this presents a correctness problem at present though, because SST files cannot have 0 size by definition, right?